From 34c09c476ca224a5141dd88740c5131907b0b8d4 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Wed, 26 Sep 2018 14:12:13 -0700
Subject: [PATCH 2/2] Use /usr/lib64

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index eb7330e..6198602 100644
--- a/Makefile
+++ b/Makefile
@@ -53,7 +53,7 @@ prefix ?= /usr/local
 exec_prefix ?= $(prefix)
 
 includedir = $(prefix)/include/mqtt
-libdir = $(exec_prefix)/lib
+libdir = $(exec_prefix)/lib64
 
 # ----- Definitions for the shared library -----
 
-- 
2.19.0

